src/refs.h

```diff
@@ -53,6 +55,7 @@
 #define GIT_REFS_STASH_FILE GIT_REFS_DIR GIT_STASH_FILE
 
 #define GIT_REF_FORMAT__PRECOMPOSE_UNICODE	(1u << 16)
+#define GIT_REF_VALIDATION_DISABLE	(1u << 15)
```

Comment:
For this one - please do use double-underscores, since it's internal to the library, and the flags are all the  `GIT_REF_FORMAT_...` constants.  So I think this would be most clear as `GIT_REF_FORMAT__VALIDATION_DISABLE`.